### PR TITLE
Move comment advert switches to commercial-features module

### PR DESCRIPTION
**Pull request by jbreckmckye**
This is another set of switches for a commercial feature that has slipped through the cracks. I've added it to commercial-features and taken the opportunity to unit test it.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/commercial-features.js [hunk 1]
```diff
@@ -102,6 +106,16 @@ define([
...
 
+        this.commentAdverts =
+            this.dfpAdvertising &&
+            switches.standardAdverts &&
+            !isMinuteArticle &&
+            config.switches.viewability &&
+            config.switches.discussion &&
+            config.page.commentable &&
+            identityApi.isUserLoggedIn() &&
+            (isLiveBlog ? isWidePage : true);
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/commercial/commercial-features.js]: `(!isLiveBlog || isWidePage)` seems slightly more idiomatic

File: static/src/javascripts/projects/common/modules/commercial/comment-adverts.js [hunk 2]
```diff
@@ -39,15 +41,7 @@ define([
...
-        if (!config.switches.standardAdverts ||
-            !config.switches.viewability ||
-            !$adSlotContainer.length ||
-            !config.switches.discussion ||
-            !identityApi.isUserLoggedIn() ||
-            (config.page.section === 'childrens-books-site' || config.page.shouldHideAdverts) || /* Sensitive pages */
-            (config.page.isLiveBlog && detect.getBreakpoint() !== 'wide') ||
-            !config.page.commentable ||
-            config.page.isMinuteArticle) {
+        if (!commercialFeatures.commentAdverts || !$adSlotContainer.length) {
```
Comment (uplne) [inline static/src/javascripts/projects/common/modules/commercial/comment-adverts.js]: This is beautiful

**Comment by regiskuckaertz** [comment]
Rookie question: why isn't this component called in commercial.js?

**Comment by jbreckmckye** [comment]
@regiskuckaertz - It's actually part of a completely separate code bundle, `trail.js`. Perhaps someone wanted to avoid adding these dependencies on pages they weren't strictly needed?

**Pull request merged**